sdk/src/client/bindings/python/iota_client/client.py

```diff
@@ -610,6 +610,39 @@ def as_dict(self):
 
         return config
 
+class Ed25519Address(Address):
+    def __init__(self, address_or_id):
+        """Initialize an Ed25519Address
+        
+        Parameters
+        ----------
+        address_or_id : string
+            The hex encoded address to use.
+        """
+        super().__init__(AddressType.ED25519, address_or_id)
```

Comment:
```suggestion
    def __init__(self, address):
        """Initialize an Ed25519Address
        
        Parameters
        ----------
        address_or_id : string
            The hex encoded address to use.
        """
        super().__init__(AddressType.ED25519, address)
```
We only have ids for alias and nft :trollface:

Reply:
How should Copilot know that? :trollface: 

```suggestion
    def __init__(self, address):
        """Initialize an Ed25519Address
        
        Parameters
        ----------
        address : string
            The hex encoded address to use.
        """
        super().__init__(AddressType.ED25519, address)
```